composer.json

```diff
@@ -1,9 +1,24 @@
 {
+  "name": "civipkg/uk.co.vedaconsulting.mosaico",
   "require": {
-    "intervention/image": "^2.4"
+    "intervention/image": "^2.4",
+    "civicrm/composer-downloads-plugin": "^3.0"
   },
   "replace": {
     "guzzlehttp/guzzle": "^6.5",
     "guzzlehttp/psr7": "^1.8.5"
+  },
+  "config": {
+    "allow-plugins": {
+      "civicrm/composer-downloads-plugin": true
+    }
+  },
+  "extra": {
+    "downloads": {
+      "mosaico": {
+        "path": "packages/mosaico",
+        "url": "https://github.com/totten/mosaico/releases/download/v0.15-civicrm-2.1/mosaico-dist.tar.gz"
```

Comment:
This pulls from a tag on `totten/mosaico`. However, if we merge https://github.com/civicrm/mosaico/pull/13 first, then we can download from `civicrm/mosaico` instead.

Reply:
@totten I merged civicrm/mosaico#13 can you update this PR now?